src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -184,6 +189,30 @@ private String commandStringGenerator(
     return commandString.toString();
   }
 
+  /**
+   * Reads the docker server configuration from file ssh.rc under "/pathToResourceDocker".
+   */
+  private void readServerConfig() {
+    String configFilePath = pathToResourceDocker + "/ssh.rc";
+    try {
+      List<String> lines = Files.readAllLines(Paths.get(configFilePath), Charset.forName("UTF-8"));
+      for (String line : lines) {
+        if (line.contains("p2p_server=") && !line.endsWith("=")) {
+          setKeyServerIp(line.split("=")[1]);
+        } else if (line.contains("http_port=") && !line.endsWith("=")) {
+          setKeyServerHttpPort(line.split("=")[1]);
+        }
```

Comment:
I just overlooked it:
What happens if the file is valid, but there were no valid options within?
From my perspective no exception gets thrown... 
Maybe I was not able to explain myself --> we could also discuss this in person :)